docs/pages/agents/deploy-agents-terraform.mdx

```diff
@@ -0,0 +1,310 @@
+configuration file to the default location, `/etc/teleport.yaml`. The
+configuration file enables each Teleport service we associated with our token.
+
+So we you access hosts in the agent pool later, the configuration also adds the
+`role: agent-pool` label to the Teleport SSH Service on each instance.
+
+Finally, the script restarts Teleport on the host to apply the new
+configuration.
+
+### Input variables
+
+The Terraform configuration we show in this guide relies on the following
+inputs:
+
+```text
+(!examples/agent-pool-terraform/inputs.tf!)
+```
+
+In your `agent-pool-terraform` project directory, create a file called
+`main.auto.tfvars` with the following content:
+
+```text
+agent_count=2
+proxy_service_address="mytenant.teleport.sh"
+aws_region=""
+teleport_version=(=teleport.version=)
+subnet_id=""
+```
+
+Assign `agent_count` to `2` for high availability. As you scale your Teleport
+usage, you can increase this count to ease the load on each agent. You can
+consider adding your agents to an Auto Scaling group as well.
+
+Assign `proxy_service_address` to the host and HTTPS port of your Teleport Proxy
+Service, e.g., `mytenant.teleport.sh:443`.
+
+<Notice type="tip">
+
+Make sure to include the port.
+
+</Notice>
+
+Assign `aws_region` to your AWS region, e.g., `us-east-1`.
+
+For `subnet_id`, include the ID of the AWS subnet where you will deploy Teleport
+agents.
+
+Finally, make sure you are using the latest supported version of the Teleport
+Terraform provider. The `required_providers` block for the Teleport provider
+includes a placeholder value:
+
+```text
+(!examples/agent-pool-terraform/provider.tf!)
+```
+
+Replace the placeholder value with the latest version:
+
+```code
+$ sed -i "" "s/TELEPORT_VERSION/(=teleport.plugin.version=)/" provider.tf
+```
+
+## Step 3/3. Verify the deployment
+
+Make sure you have made your AWS credentials available to Terraform using the
+standard approach for your organization.
+
+Apply the Terraform configuration:
+
+```code
+$ terraform apply
+```
+
+Once the `apply` command completes, run the following command to verify that the
+two agents have deployed successfully:
+
+```code
+$ tsh ls role=agent-pool
+Node Name                  Address    Labels
+-------------------------- ---------- ---------------
+ip-10-1-1-187.ec2.internal ⟵ Tunnel   role=agent-pool
+ip-10-1-1-24.ec2.internal  ⟵ Tunnel   role=agent-pool
+```
+
+## Next step: Enroll infrastructure resources
+
+There are two ways to configure your agent pool to protect infrastructure
+resources with Teleport:
+
+- **Dynamic infrastructure resources:** You can declare additional Terraform
+  resources that represent resources in your infrastructure. Teleport agents
+  look up dynamic infrastructure resources from the Auth Service in order to
+  proxy user traffic.
+- **Static configuration settings:** Edit the configuration of each service to
+  point to specific resources in your infrastructure.
+
```

Comment:
Suggest a new paragraph here pointing to a page for configuring static settings and declaring that the next section of the guide will cover dynamic resources.

Reply:
Or, is the "Configure Teleport services in the agent pool" section is the counter-point to the "Define dynamic resources in Terraform" section, suggest rephrasing the heading to better clarify that each of the sections relates to each of these bullet points.

Reply:
I've removed the bulleted list to go straight into the H3s—hopefully that clears things up